app/js/shipyard/factory-ship.js

```diff
@@ -548,7 +548,12 @@ angular.module('shipyard').factory('Ship', ['Components', 'calcShieldStrength',
   };
 
   Ship.prototype.useUtility = function(group, rating, clobber) {
-    var component = Components.findHardpoint(group, 0, rating);
+    var component;
+    if (group == 'cm') {
+      component = Components.findHardpoint(group, 0, null, rating); //overload Rating as Name for countermeasures
```

Comment:
Yep they're optional params anyway, so no harm in extending; I'll make the change right now.